src/utils/messageExtension.ts

```diff
@@ -18,6 +18,14 @@ export const messageExtension = {
       return false;
     }
   },
+  isBotWelcomeMsg(message: BaseMessage, botId: string | null) {
+    if ((message.isUserMessage() || message.isFileMessage()) && message.sender.userId === botId) {
+      const data = parseMessageDataSafely(message.data);
+      return !data?.respond_mesg_id && !data?.stream;
```

Comment:
can we add note on why we need `!data?.respond_mesg_id && !data?.stream;`? the function name seems unrelated

Reply:
addressed in https://github.com/sendbird/chat-ai-widget/pull/312/commits/ddc1dbd263cb8a49925047b53a398eb7ce117f9e